docker-compose.yml

```diff
@@ -0,0 +1,76 @@
+version: '3'
+
+services:
+    catalog:
+        build:
+            context: catalog
+        ports:
+            - "5430:5432"
```

Comment:
Leave a comment explaining that we have to expose a different port due to the `metalnx-db` service. Eventually that service will go away and with it we can remove this stanza.